sirepo/package_data/template/flash/lib/CapLaser.setup_units

```diff
@@ -0,0 +1,161 @@
+Driver
```

Comment:
@robnagler do we want this file here? We parse it [when generating parameters.py for export](https://github.com/radiasoft/sirepo/blob/47a24d8bd21f8916dddad2146a0106544b5924bd/sirepo/template/flash.py#L427). Maybe it should be packaged with the binary and distributed with it?